src/client/reducers/veteran/index.js

```diff
@@ -141,133 +141,443 @@ const blankVeteran = {
-  cityOfBirth: makeField('Springfield', true),
-  stateOfBirth: makeField('AK', true),
+    first: {
+      value: 'FirstName',
+      dirty: true
+    },
+    middle: {
+      value: 'MiddleName',
+      dirty: true
+    },
+    last: {
+      value: 'LastName',
+      dirty: true
+    },
+    suffix: {
+      value: 'Jr.',
+      dirty: true
+    }
+  },
+  mothersMaidenName: {
+    value: 'Maiden',
+    dirty: true
+  },
+  veteranSocialSecurityNumber: {
+    value: '111-11-1234',
+    dirty: true
+  },
+  gender: {
+    value: 'F',
+    dirty: true
+  },
+  cityOfBirth: {
+    value: 'Springfield',
+    dirty: true
+  },
+  stateOfBirth: {
+    value: 'AK',
+    dirty: true
+  },
   veteranDateOfBirth: {
-    month: makeField('1', true),
-    day: makeField('2', true),
-    year: makeField('1923', true),
+    month: {
+      value: '1',
+      dirty: true
+    },
+    day: {
+      value: '2',
+      dirty: true
+    },
+    year: {
+      value: '1923',
+      dirty: true
+    }
   },
-  maritalStatus: makeField('Married'),
-
-  isVaServiceConnected: makeField(''),
-  compensableVaServiceConnected: makeField(''),
-  receivesVaPension: makeField(''),
-
-  isEssentialAcaCoverage: false,
-  facilityState: makeField(''),
-  vaMedicalFacility: makeField(''),
-  wantsInitialVaContact: false,
-
-  isSpanishHispanicLatino: false,
-  isAmericanIndianOrAlaskanNative: false,
-  isBlackOrAfricanAmerican: false,
-  isNativeHawaiianOrOtherPacificIslander: false,
-  isAsian: false,
-  isWhite: false,
-
+  maritalStatus: {
+    value: 'Married',
+    dirty: false
+  },
+  isVaServiceConnected: {
+    value: 'N',
+    dirty: true
+  },
+  compensableVaServiceConnected: {
+    value: 'N',
+    dirty: true
+  },
+  receivesVaPension: {
+    value: 'Y',
+    dirty: true
+  },
+  isEssentialAcaCoverage: true,
+  facilityState: {
+    value: 'CT',
+    dirty: true
+  },
+  vaMedicalFacility: {
+    value: '689A4',
+    dirty: true
+  },
+  wantsInitialVaContact: true,
+  isSpanishHispanicLatino: true,
+  isAmericanIndianOrAlaskanNative: true,
+  isBlackOrAfricanAmerican: true,
+  isNativeHawaiianOrOtherPacificIslander: true,
+  isAsian: true,
+  isWhite: true,
   veteranAddress: {
-    street: makeField('123 NW 5th St'),
-    city: makeField('Washington'),
-    country: makeField('USA'),
-    state: makeField('DC'),
-    zipcode: makeField('001234-1234'),
+    street: {
+      value: '123 NW 5th St',
+      dirty: false
+    },
+    city: {
+      value: 'Washington',
+      dirty: false
+    },
+    country: {
+      value: 'USA',
+      dirty: false
+    },
+    state: {
+      value: 'DC',
+      dirty: false
+    },
+    zipcode: {
+      value: '001234-1234',
+      dirty: false
+    }
+  },
+  veteranCounty: {
+    value: 'USA',
+    dirty: true
+  },
+  email: {
+    value: 'foo@example.com',
+    dirty: false
+  },
+  emailConfirmation: {
+    value: 'foo@example.com',
+    dirty: false
+  },
+  homePhone: {
+    value: '123-124-1234',
```

Comment:
The validations for phone number have changed so that it no longer accepts dashes.